package.json

```
@@ -306,7 +310,7 @@
 			"node ./docs/tool/index.js"
 		],
 		"packages/**/*.{js,ts,tsx}": [
```

We need to include JSON files as well 😄 
```suggestion
		"packages/**/*.{js,ts,tsx,json}": [
```

---

😁